### DISCUSSION TITLE
I have an issues when installing foundry

### DISCUSSION THREAD

**Discussion by chitu-hou** [Cyfrin/foundry-full-course-cu#1791] [Q&A]
When i input this on my  terminal
curl -L https://foundry.paradigm.xyz | bash
I always get this

[wode@localhost ~]$ curl -L https://foundry.paradigm.xyz | bash
 % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   167  100   167    0     0    166      0  0:00:01  0:00:01 --:--:--   166
  0     0    0     0    0     0      0      0 --:--:--  0:00:01 --:--:--     0
curl: (51) Unable to communicate securely with peer: requested domain name does not match the server's certificate.

**Comment by EngrPips** [comment]
Hello @chitu-hou, I haven't run into this error before so I am unsure how to solve this. But this is `https://stackoverflow.com/questions/60644617/curl-says-requested-domain-name-does-not-match-the-servers-certificate-but-i`  what I found when I looked it up. Maybe take a look and see if it will help at all.
